config/config_test.go

```diff
@@ -1024,10 +1031,10 @@ var expectedErrors = []struct {
 		errMsg:   "invalid selector: 'metadata.status-Running'; can't understand 'metadata.status-Running'",
 	}, {
 		filename: "kubernetes_bearertoken_basicauth.bad.yml",
-		errMsg:   "at most one of basic_auth, bearer_token & bearer_token_file must be configured",
+		errMsg:   "at most one of basic_auth, oauth2, bearer_token & bearer_token_file must be configured",
 	}, {
 		filename: "kubernetes_authorization_basicauth.bad.yml",
-		errMsg:   "at most one of basic_auth & authorization must be configured",
+		errMsg:   "at most one of basic_auth, oauth2 & authorization must be configured",
```

Comment:
```suggestion
		errMsg:   "at most one of basic_auth, oauth2, & authorization must be configured",
```

Reply:
That's not the error message returned; it breaks tests.

https://github.com/prometheus/common/blob/b4304c504af1cf993f7b6873c5455e0ac51726a7/config/http_config.go#L195

Reply:
That's fine then. I think we could add the oxford comma next time.

Reply:
This one doesn't have an oxford comma either. Not sure if that's my doing or if it was already there.

https://github.com/prometheus/common/blob/b4304c504af1cf993f7b6873c5455e0ac51726a7/config/http_config.go#L175